cmd/zetaclientd/keygen_tss.go

```diff
@@ -16,9 +17,9 @@ import (
 	"gitlab.com/thorchain/tss/go-tss/p2p"
 )
 
-func GenerateTss(logger zerolog.Logger, cfg *config.Config, zetaBridge *mc.ZetaCoreBridge, peers p2p.AddrList, priKey secp256k1.PrivKey, ts *mc.TelemetryServer) (*mc.TSS, error) {
+func GenerateTss(logger zerolog.Logger, cfg *config.Config, zetaBridge *mc.ZetaCoreBridge, peers p2p.AddrList, priKey secp256k1.PrivKey, ts *mc.TelemetryServer, tssList []*types.TSS) (*mc.TSS, error) {
```

Comment:
```
tssList []*types.TSS
```
Is it the TSS history? I would rename it `tssHistoricalList` to be more explicit